cmd/project.go

```diff
@@ -36,22 +36,50 @@ Use the subcommands to manage your projects.`,
 	},
 }
 
-func saveEnv(dir string, apikey string) {
+func saveEnv(dir string, apikey string, projectKey string) {
 	filename := filepath.Join(dir, ".env")
 	envLines, err := env.ParseEnvFile(filename)
 	if err != nil {
 		errsystem.New(errsystem.ErrReadConfigurationFile, err, errsystem.WithContextMessage("Failed to parse .env file")).ShowErrorAndExit()
 	}
-	var found bool
+	var found map[string]bool = map[string]bool{
+		"AGENTUITY_SDK_KEY":     false,
+		"AGENTUITY_API_KEY":     false,
+		"AGENTUITY_PROJECT_KEY": false,
+	}
+
 	for i, envLine := range envLines {
 		if envLine.Key == "AGENTUITY_API_KEY" {
 			envLines[i].Val = apikey
-			found = true
+			found["AGENTUITY_API_KEY"] = true
+		}
+		if envLine.Key == "AGENTUITY_SDK_KEY" {
+			envLines[i].Val = apikey
+			found["AGENTUITY_SDK_KEY"] = true
+		}
+		if envLine.Key == "AGENTUITY_PROJECT_KEY" {
+			envLines[i].Val = projectKey
+			found["AGENTUITY_PROJECT_KEY"] = true
+		}
```

Comment:
you can reduce the likelyhood of a fat-finger mistake
```suggestion
		if envLine.Key == "AGENTUITY_SDK_KEY" {
			envLines[i].Val = apikey
			found[envLine.Key] = true
		}
		if envLine.Key == "AGENTUITY_PROJECT_KEY" {
			envLines[i].Val = projectKey
			found[envLine.Key] = true
		}
```